.github/ariane-config.yaml

```diff
@@ -97,42 +97,42 @@ triggers:
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-aws-cni.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-clustermesh.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-delegated-ipam.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-ipsec-e2e.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-eks.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-externalworkloads.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-gateway-api.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-ginkgo.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|cilium-cli|Documentation|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|cilium-cli|Documentation)/|(CODEOWNERS|USERS.md)$)
   conformance-gke.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-ingress.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-multi-pool.yaml:
-    paths-ignore-regex: (bpf/complexity-tests|bpf/tests|test|Documentation|hubble|CODEOWNERS|USERS.md)/
+    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|test|Documentation|hubble)/|(CODEOWNERS|USERS.md|.*_test\.go)$)
   conformance-runtime.yaml:
```

Comment:
Don't the runtime tests include a run of privileged tests stored in `_test.go`?

Reply:
https://github.com/cilium/cilium/blob/5d0e9d3933c9fc90a951ef385f2ea58c609eb1f0/.github/workflows/conformance-runtime.yaml#L379-L395

Reply:
Yes, and indeed they are not skipped in that case:

```yaml
conformance-runtime.yaml:
    paths-ignore-regex: ((bpf/complexity-tests|bpf/tests|cilium-cli|Documentation)/|(CODEOWNERS|USERS.md)$)
```

Reply:
:facepalm: Sorry for the noise, I slipped in the line